frontend/dance-with-me/src/components/Landing/Landing.jsx

```diff
@@ -42,7 +41,7 @@ function Landing() {
       alt="First slide"
     />
     <Carousel.Caption>
-      <h3><Login2/></h3>
+      <h3><Login text="Find your matches" color="primary"/></h3>
```

Comment:
@u-uysal would you please make the first characters caps lock "Find Your Matches" ?

Reply:
@Mert1980 Ok. You are right. I think it looks nicer this way. Thanks to your feedback